homeassistant/components/unifi/sensor.py

```diff
@@ -185,3 +236,172 @@ async def options_updated(self) -> None:
         """Config entry options are updated, remove entity if option is disabled."""
         if not self.controller.option_allow_uptime_sensors:
             await self.remove_item({self.client.mac})
+
+
+@dataclass
+class UniFiDeviceSensorEntityDescription(SensorEntityDescription):
+    """Describes UniFi device sensor entities."""
+
+    enabled: str | None = None
+    value_fn: Callable[[UniFiDevice], Any] | None = None
+
+    def get_value(self, device: UniFiDevice) -> Any:
```

Comment:
Inexplicit type, this needs to be changed to explicit else I will need to rework stuff over time.

Reply:
I can replace `Any` with a union of all of the return types of the `lambda`s, but that means whenever a new sensor is added the type may need to be updated if the new `lambda` returns some other type.

I might be missing something or misunderstanding you

Reply:
Yes, I think that could do it, there are only like 7 types supported in sensor platform.

None, datetime.date, datetime.datetime, decimal.Decimal, float, int, string